docs/app/experiments/accordion.tsx

```diff
@@ -0,0 +1,194 @@
+            This is the contents of Accordion.Panel 3
+          </Accordion.Panel>
+        </Accordion.Item>
+
+        <Accordion.Item className="MyAccordion-item">
+          <Accordion.Header className="MyAccordion-header">
+            <Accordion.Trigger className="MyAccordion-trigger">Trigger 4</Accordion.Trigger>
+          </Accordion.Header>
+          <Accordion.Panel className="MyAccordion-panel">
+            This is the contents of Accordion.Panel 4
+          </Accordion.Panel>
+        </Accordion.Item>
+
+        <Accordion.Item className="MyAccordion-item">
+          <Accordion.Header className="MyAccordion-header">
+            <Accordion.Trigger className="MyAccordion-trigger">Trigger 5</Accordion.Trigger>
+          </Accordion.Header>
+          <Accordion.Panel className="MyAccordion-panel">
+            This is the contents of Accordion.Panel 5
+          </Accordion.Panel>
+        </Accordion.Item>
+      </Accordion.Root>
+
+      <hr style={{ marginTop: 32, marginBottom: 32 }} role="presentation" />
+
+      <h2>Controlled</h2>
+
+      <Accordion.Root
+        className="MyAccordion-root"
+        value={val}
+        onOpenChange={setVal}
+        aria-label="Controlled Accordion"
+        openMultiple={openMultiple}
+      >
+        <Accordion.Item className="MyAccordion-item" value="one">
+          <Accordion.Header className="MyAccordion-header">
+            <Accordion.Trigger className="MyAccordion-trigger">Trigger 1</Accordion.Trigger>
+          </Accordion.Header>
+          <Accordion.Panel className="MyAccordion-panel">
+            This is the contents of Accordion.Panel 1, the value is &quot;one&quot;
+          </Accordion.Panel>
+        </Accordion.Item>
+
+        <Accordion.Item className="MyAccordion-item" value="two">
+          <Accordion.Header className="MyAccordion-header">
+            <Accordion.Trigger className="MyAccordion-trigger">Trigger 2</Accordion.Trigger>
+          </Accordion.Header>
+          <Accordion.Panel className="MyAccordion-panel">
+            This is the contents of Accordion.Panel 2, the value is &quot;two&quot;
+          </Accordion.Panel>
+        </Accordion.Item>
+
+        <Accordion.Item className="MyAccordion-item" value="three">
+          <Accordion.Header className="MyAccordion-header">
+            <Accordion.Trigger className="MyAccordion-trigger">Trigger 3</Accordion.Trigger>
+          </Accordion.Header>
+          <Accordion.Panel className="MyAccordion-panel">
+            This is the contents of Accordion.Panel 3, the value is &quot;three&quot;
+          </Accordion.Panel>
+        </Accordion.Item>
+      </Accordion.Root>
+
+      <hr style={{ marginTop: 32, marginBottom: 32 }} role="presentation" />
+
+      <h2>Controlled, at least one section must remain open</h2>
+
+      <Accordion.Root
+        className="MyAccordion-root"
+        value={val2}
+        onOpenChange={(newValue: Accordion.Root.Props['Value']) => {
```

Comment:
Yeah I'd expect `onValueChange` here too

Reply:
👌 I've changed `onOpenChange` to `onValueChange` ~

Reply:
> Having `expanded` seems closer to what this do?

@oliviertassinari ~ makes sense and chatted with the team about this, in the end we decided:
- for Accordion, use `value` since there can be multiple values (multiple open sections), and `open`/`expanded` sounds too much like a boolean prop (much like our other components that use `open`)
- for Collapsible, stick with `open` rather `expanded`, as `expanded`/`onExpandedChange` makes the JS API sound a bit clumsy 😅 (we also don't need to introduce `expanded` into the existing vocabulary)